packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/ReactTextPropertiesResolverTest.kt

```diff
@@ -0,0 +1,251 @@
+            drawableUtils = mockDrawableUtils
+        )
+    }
+
+    @Test
+    fun `M get original wireframe W addReactNativeProperties() { no react properties }`() {
+        // Given
+        whenever(mockTextView.background).thenReturn(null)
+
+        // When
+        val result = testedResolver.addReactNativeProperties(mockWireframe, mockTextView, 0f)
+
+        // Then
+        assertThat(result).isEqualTo(mockWireframe)
+    }
+
+    @Test
+    fun `M add drawable properties W addReactNativeProperties() { has reactBackgroundDrawable }`(
+        forge: Forge
+    ) {
+        // Given
+        val pixelDensity = 0f
+        val fakeBorderRadius = forge.aPositiveFloat()
+        val fakeBorderWidth = forge.aPositiveFloat()
+        val fakeBorderColor = forge.aPositiveInt()
+        whenever(mockTextView.background).thenReturn(mockReactViewBackgroundDrawable)
+        whenever(
+            mockDrawableUtils.getReactBackgroundFromDrawable(
+                eq(mockTextView.background)
+            )
+        ).thenReturn(mockReactViewBackgroundDrawable)
+        whenever(
+            mockReactViewBackgroundDrawableUtils.resolveShapeAndBorder(
+                drawable = eq(mockReactViewBackgroundDrawable),
+                opacity = eq(0f),
+                pixelDensity = eq(0f)
+            )
+        ).thenReturn(
+            MobileSegment.ShapeStyle(
+                backgroundColor = formatAsRgba(fakeBorderColor),
+                opacity = 0f,
+                cornerRadius = fakeBorderRadius.toLong()
+            ) to MobileSegment.ShapeBorder(
+                color = formatAsRgba(fakeBorderColor),
+                width = fakeBorderWidth.toLong()
+            )
+        )
+
+        // When
+        val result = testedResolver.addReactNativeProperties(
+            fakeWireframe,
+            mockTextView,
+            pixelDensity
+        )
+
+        // Then
+        assertThat(result.shapeStyle?.cornerRadius).isEqualTo(fakeBorderRadius.toLong())
+        assertThat(result.border?.width).isEqualTo(fakeBorderWidth.toLong())
+        assertThat(result.border?.color).isEqualTo(formatAsRgba(fakeBorderColor))
+    }
+
+    // region addReactNativeProperties
+
+    @Test
+    fun `M resolve font family W addReactNativeProperties()`() {
+        // Given
+        whenever(mockShadowNodeWrapper.initialize(mockTextView.id)).thenReturn(true)
+        whenever(mockTextView.background).thenReturn(null)
+        whenever(mockShadowNodeWrapper.getDeclaredShadowNodeField(FONT_FAMILY_FIELD_NAME))
+            .thenReturn(ReactTextPropertiesResolver.MONOSPACE_FAMILY_NAME)
+
+        // When
+        val result = testedResolver
+            .addReactNativeProperties(fakeWireframe, mockTextView, 0f)
+
+        // Then
+        assertThat(result.textStyle.family)
+            .isEqualTo(ReactTextPropertiesResolver.MONOSPACE_FAMILY_NAME)
+    }
+
+    @Test
+    fun `M fallback W addReactNativeProperties() { cant resolve fontFamily }`() {
```

Comment:
```suggestion
    fun `M fallback W addReactNativeProperties() { cannot resolve fontFamily }`() {
```

same for methods below